osbs/constants.py

```diff
@@ -76,6 +76,9 @@
 ATOMIC_REACTOR_LOGGING_FMT = \
     '%(asctime)s platform:%(arch)s - %(name)s - %(levelname)s - %(message)s'
 
+# Numeric representation of user warnings loggers' level
+USER_WARNING_LEVEL = 25
```

Comment:
It would be good to give an informative description of number 25.

Reply:
Added